src/Shared/Traits.cs

```diff
@@ -115,6 +115,11 @@ internal class EscapeHatches
         /// </summary>
         public readonly bool AlwaysUseContentTimestamp = Environment.GetEnvironmentVariable("MSBUILDALWAYSCHECKCONTENTTIMESTAMP") == "1";
 
+        /// <summary>
+        /// Stop truncating task inputs when logging them.
+        /// </summary>
+        public readonly bool TruncateTaskInputLogging = Environment.GetEnvironmentVariable("MSBUILDTRUNCATETASKINPUTLOGGING") == "1";
```

Comment:
https://github.com/microsoft/msbuild/pull/5210#discussion_r401197592

Reply:
I opted for a name change to make it easier to understand. But I realize we could use the same name and have the condition be something like this: `public readonly bool LogAllTaskInputs = string.IsNullOrEmpty(Environment.GetEnvironmentVariable("MSBUILDLOGALLTASKINPUTS"));` so if it is empty (as it should be by default), it'll log everything. To disable it you'd have to set it to any value. It makes slightly less sense to me this way but that seems to be the convention.

Reply:
After changing it back to `LogAllTaskInputs`, I reverted back to the `TruncateTaskInputs` convention. It just makes more sense to me.